### PR TITLE
ARROW-3521: [GLib] Run Python using find_program in meson.build

### PR DESCRIPTION
**Pull request by shiro615**
Because it is possible to run “python3” or “python2".

### CODE REVIEW

File: c_glib/meson.build [hunk 1]
```diff
@@ -23,7 +23,8 @@ project('arrow-glib', 'c', 'cpp',
           'cpp_std=c++11',
         ])
 
-version = run_command('python', 'tool/get-version.py').stdout().strip()
+python = find_program('python3', 'python', 'python2')
```
Comment (xhochy) [inline c_glib/meson.build]: Is the order here also the preference used? Normally `python` is the preferred one when you're in `virtualenv` or a `conda` env. Thus as a Python developer, I would like always the plain `python` executable one as the preferred.

Comment (kou) [reply c_glib/meson.build]: OK. Let's use `'python', 'python3', 'python2'` order.

Comment (shiro615) [reply c_glib/meson.build]: I've changed the order.

**Review by kou — APPROVED** [review]
+1
Thanks!

**Pull request closed without merging**